spec/controllers/users/sessions_controller_spec.rb

```diff
@@ -30,6 +31,116 @@
           post :create, params: {user: {email: user.email, password: user.password}}
         end.to raise_error(ActionController::UnknownFormat)
       end
+
+      it "returns auth failed when user param is missing" do
```

Comment:
Can you test the situation where a user's account is locked? I want to make sure that still works as expected.